README.md

```
@@ -1,3 +1,5 @@
+### Assignment 1 By Divyansh Jain
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)